dashboards/defaults.libsonnet

```diff
@@ -0,0 +1,13 @@
+{
+  local grafanaDashboards = super.grafanaDashboards,
+
+  // Automatically add a uid to each dashboard based on the base64 encoding
```

Comment:
Is this something suggested by k8s or we are framing this rule?

Reply:
This is similar to the K8s

Reply:
Ack